frontend/src/lib/action.ts

```diff
@@ -0,0 +1,46 @@
+import { mockAccount, mockClosedOrders, mockOpenorders } from "./mock";
+
+export async function getClosedOrders() {
+  if (import.meta.env.MODE === "development") {
+    return mockClosedOrders;
+  } else {
+    try {
+      const response = await fetch("http://localhost:5000/orders-closed");
+      const data = await response.json();
+      return data.orders;
+    } catch (error) {
+      console.log(error);
```

Comment:
Eventually, lets make a toast on the UI that tells the customer about our error.

Reply:
Agreed